app/src/main/java/com/willowtree/vocable/settings/EditPhrasesFragment.kt

```diff
@@ -56,8 +56,7 @@ class EditPhrasesFragment : BaseFragment() {
             val phraseView =
                 PhraseEditLayoutBinding.inflate(inflater, binding?.editPhrasesContainer, false)
             with(phraseView.root) {
-                // TODO: Use currently set Locale
-                phrase_edit_text.text = phrase.localizedUtterance[Locale.US.language]
+                phrase_edit_text.text = phrase.getLocalizedText()
```

Comment:
Not sure if you've already re-factored some of this out but try to avoid Kotlin synthetics! I thought I saw binding somewhere before, didn't yall use view binding?

Reply:
Yeah we've moved the app over to binding, so this one must have slipped through. Nice catch!